### ISSUE TITLE
Load plugins from json config instead of iterating directory

### ISSUE THREAD

**Issue by ribtoks** [ribtoks/xpiks#753]
part of #445 effort
dependent on #752 

This task is about loading plugins only from json file instead of iterating directory `/plugins/`. Json config from #752 should contain path to main plugin file and Xpiks should only load plugin from this path.

Currently all plugins that fail to load during directory iteration are put to smth like `/failed_plugins/` directory. After adding loading from json this must be deprecated and they should be kept where they are now. Only thing is they should be marked "errored" in the json config and not loaded again.

There's another task to display load errors in the UI.
